v1/plugin/bool_test.go

```diff
@@ -31,83 +31,74 @@ import (
 func TestBoolRule(t *testing.T) {
 	ts := createPassTestCases()
 	Convey("Test boolRule Passes", t, func() {
-		var br boolRule
+		p := NewConfigPolicy()
 		var err error
 		for _, c := range ts {
 			Convey(fmt.Sprintf("Test boolRule %+v", c.input.key), func() {
 				if c.input.opts != nil {
-					br, err = NewBoolRule(c.input.key, c.input.req, c.input.opts)
+					err = p.AddNewBoolRule(c.input.ns, c.input.key, c.input.req, c.input.opts)
 				} else {
-					br, err = NewBoolRule(c.input.key, c.input.req)
+					err = p.AddNewBoolRule(c.input.ns, c.input.key, c.input.req)
```

Comment:
Same comment than above https://github.com/intelsdi-x/snap-plugin-lib-go/pull/31#discussion_r81186810

Those tests have been moved to `config_policy_test.go`.